### DISCUSSION TITLE
Too wide

### DISCUSSION THREAD

**Discussion by Yavinah** [YU000jp/logseq-page-tags-and-hierarchy#6] [Ideas]
Hello and first of all: Thank you for this great plugin!

It would be nice to have the option to make the tagged pages and hierarchy area smaller. Because in my case the names of my tagged pages are only 10-15 characters long, so the plugin takes unnecessarily much space from my screen.

Thank you!

**Comment by YU000jp** [comment]
Indeed, you were absolutely correct. Particularly regarding laptops, it seems that the space was indeed too large. Hence, we have made modifications this time to decrease the space in accordance with the window size and have released a new version. Please try out it. If you still find that the sizing doesn't quite fit, please let us know.

**Reply by Yavinah** [reply]
- It's much better now, but there's still a problem with the text links when they go over more than one line (see screenshot).  I use a 12" laptop and a 24" computer, so it would be much nicer to have the addon scale in percentages rather than pixels (or an option to change the width by myself).
- When the screen is too small and the addon moves down, the addon window covers a lot of my content on that page, so I always need the addon window on the right side. Maybe an option could be set up to disable the addon at the bottom so that it only appears on the right side if there is enough space, and if not it disappears?

**Reply by Yavinah** [reply]
![Screenshot 2023-06-11 103255](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/104633946/adab92c8-16b7-4f01-8c97-03c6ae69820c)

Forgot to link the screenshot :-)

**Reply by YU000jp** [reply]
I have incorporated several suggestions. In the case of smaller sizes, I have discontinued displaying it at the bottom. I plan to add that option in the future.
I have confirmed the reproduction of a bug that spans multiple lines and have found a solution for it.
